updates/marketplace-updates/may-31st-2024.md

```diff
@@ -0,0 +1,50 @@
+# May 31st, 2024
+
+Explore the new changes to the Marketplace in the last week!
+
+This can be anything from new crates, enhancements, or bug fixes!
+
+<details>
+
+<summary><strong>New crates and enhancements</strong></summary>
+
+* Rewst: User Onboarding
+  * Changed manual license approval to CWM ticket link as default behavior when `use_pod_notifications` org variable is removed.
```

Comment:
Changed manual license approval to CWM ticket link as default behavior. To use pod notifications, set the `use_pod_notifications` org variable to `true`.